src/sortStudents.ts

```diff
@@ -1,16 +1,67 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'Name',
+  Surname = 'Surname',
+  Age = 'Age',
+  Married = 'Married',
+  AverageGrade = 'AverageGrade'
```

Comment:
```suggestion
  Name,
  Surname,
  Age,
  Married,
  AverageGrade
```

but you can use here lowercase and combine logic for sorting by fields with same data types using `user[sortBy]`

```suggestion
  Name = 'name',
  Surname = 'surname',
  Age = 'age',
  Married = 'married',
  AverageGrade = 'grades'
```